src/components/BookComponent/AdBook.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { useDispatch } from 'react-redux';
+import { v4 as uuidv4 } from 'uuid';
+import { AddBook } from '../../redux/books/books';
+import styles from './AdvBook.module.css';
+
+function AddMyBookForm() {
+  const dispatchBook = useDispatch();
+  const [singleBook, setSingleBook] = useState({
+    title: '',
+    category: '',
+    author: '',
+    item_id: '',
+  });
+
+  // Changing state
+  const changeFunctio = (e) => {
+    const [name, value] = [e.target.name, e.target.value];
+    if (value.trim() === '') e.target.value = '';
+
+    setSingleBook((previous) => ({
+      ...previous,
+      [name]: value.trim(),
+    }));
+  };
+
+  // submit
+  const submitMyForm = (e) => {
+    e.preventDefault();
+    dispatchBook(AddBook({ ...singleBook, item_id: uuidv4() }));
+    document.querySelector('.form-input').reset();
+  };
+  return (
+    <div className={styles.containers}>
+      <h1>ADD NEW BOOK</h1>
+      <form name="addBook" onSubmit={submitMyForm} className="form-input">
+        <input type="text" name="title" placeholder="Book title" onChange={changeFunctio} required />
+        <input type="text" name="author" placeholder="Author name" onChange={changeFunctio} required />
+        <select name="category" onChange={changeFunctio} required>
+          <option value="" defaultValue>category</option>
+          <option value="Science">Science</option>
+          <option value="Art">Art</option>
+          <option value="Commercial">Commercial</option>
+        </select>
+        <input className={styles.butt} type="submit" value="ADD BOOK" />
+      </form>
+    </div>
```

Comment:
- [ ] Please ensure that your input elements are styled as seen in the Figma design and the `Add new book` section needs to be the same width as the book cards as seen in the design so kindly implement this while perfecting your design :100: 
- Your design
![image](https://github.com/citec-47/bookstoreReact/assets/102699958/ad3c6cbd-8bd8-4fbe-993e-053c5671092c)

- The Zeplin design
![image](https://github.com/citec-47/bookstoreReact/assets/102699958/442863d2-21d9-4943-bd5e-4d1dbc7884b1)